app/models/account/states.rb

```diff
@@ -138,11 +138,15 @@ def will_be_deleted?
       scheduled_for_deletion? || (buyer? && provider_account.try(:should_be_deleted?))
     end
 
-    def can_be_suspended?
+    def ready_to_be_suspended?
```

Comment:
Usually is better to use active instead of passive voice. Here `ready_to_suspend?` would be a better method name